### PR TITLE
Added Thaniel's feature

### PR DESCRIPTION
**Pull request by ThanielPri**
I wrote down every markdown I saw from the lab slide. Please merge it with master

### CODE REVIEW

File: workflow.md [hunk 1]
```diff
@@ -18,6 +18,16 @@ You all preform a pull request if you would have any problems, or if you want me
...
 ### 2. Thaniel's feature
+# A header  
+`emphasis`
+*This is a sentence with star*  
+** no star _some star with underline_**
+~~this sentence is dicoret~~  
+
+1. Numero uno
+2. Numero dos  
+
```
Comment (michieldevos) [inline workflow.md:29]: Could you add a number 3 as well? I think the program will run better then.

**Comment by michieldevos** [comment]
Hi Thaniel, thank you. Could you check the above comment, to resolve the latest bugs

File: workflow.md (hunk 1 above)
Comment (ThanielPri) [reply workflow.md:30]: Yes, I have added a number 3 on the feature I am working on.

**Comment by michieldevos** [comment]
Looks great, I tested it and it works. I will implement it.

**Pull request merged**